5_recognition.ipynb

```diff
@@ -368,25 +652,58 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "このxyz値は、「head_rgbd_sensor_rgb_frame」基準座標、つまりRGBDカメラ座標上での値です。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "This xyz value is the value on the \"head_rgbd_sensor_rgb_frame\" reference coordinates, that is the RGBD camera coordinates."
```

Comment:
"This xyz value is the value in the \"head_rgbd_sensor_rgb_frame\" reference coordinates, that is the RGBD camera coordinates."